source/simulator/parameters.cc

```diff
@@ -739,6 +739,28 @@ namespace aspect
       prm.declare_entry ("Names of fields", "",
                          Patterns::List(Patterns::Anything()),
                          "A user-defined name for each of the compositional fields requested.");
+      prm.declare_entry ("Compositional field methods", "",
+                         Patterns::List (Patterns::Selection("continuous field|particles")),
+                         "A comma separated list denoting the solution method of each "
+                         "compositional field. Each entry of the list must be "
+                         "one of the currently implemented field types: "
+                         "``continuous_fem_field'', ``discontinuous_fem_field'', "
+                         "or ``particles''.");
+      prm.declare_entry ("Mapped particle properties", "",
+                         Patterns::Map (Patterns::Anything(),
+                                        Patterns::Anything()),
+                         "A comma separated list denoting the particle properties "
+                         "that will be projected to those compositional fields that "
+                         "are of the ``particles'' field type."
+                         "\n\n"
+                         "The format of valid entries for this parameter is that of a map "
+                         "given as ``key1: value1, key2: value2 [component2], key3: value3 [component4], "
+                         "...'' where each key must be a valid field name of the "
+                         "``particles'' type, and each value must be one of the currently "
+                         "selected particle properties. Component is a component index of "
+                         "the particle property that is 0 by default, and only needs to be "
+                         "set if a different component of the particle property should be "
+                         "mapped (e.g. the y-component of the velocity at the particle positions).");
```

Comment:
-> `$y$`

I think what the documentation is lacking is a half-sentence that indicates that particle property names may imply multiple properties (e.g., the particle position implies a *vector* of properties), and that the component selector picks one of those if there are in fact multiple.